smart_contracts/contract/src/contract_api/runtime.rs

```diff
@@ -404,6 +405,44 @@ pub fn get_call_stack() -> Vec<CallStackElement> {
     bytesrepr::deserialize(bytes).unwrap_or_revert()
 }
 
+/// Manages a message topic.
+pub fn manage_message_topic(
+    topic_name: &str,
+    operation: MessageTopicOperation,
+) -> Result<(), ApiError> {
+    if topic_name.is_empty() {
+        return Err(ApiError::InvalidArgument);
+    }
+
+    let (topic_name_ptr, topic_name_size, _bytes) = contract_api::to_ptr(topic_name);
```

Comment:
See my other comment regarding passing strings

Reply:
Changed in https://github.com/casper-network/casper-node/commit/53483e16590b67de9be9be4d9ba4e6b64d1f22ee